### PR TITLE
Add v1alpha3 CAPBM version

### PR DESCRIPTION
**Pull request by maelk**
and change v1alpha2 CAPBM to release branch

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -26,6 +26,9 @@ test:
 test_v1a2:
 	./05_test_v1a2.sh
 
+test_v1a3:
+	./05_test_v1a2.sh
```
Comment (Jaakko-Os) [inline Makefile]: Script name should not be version specific if used like this.

Comment (kashifest) [reply Makefile]: Is it a typo or are we using v1a2 test scripts for v1a3 also?

Comment (maelk) [reply Makefile]: This is just to keep the CI jobs green. Once the v1a3 target exists, we'll change it

**Pull request merged**